docs/user_guide/spin_system/spin_system.rst

```diff
@@ -4,10 +4,56 @@
 Spin System
 ===========
 
-At the heart of any mrsimulator calculation is the definition of a :ref:`spin_sys_api`
-object describing the sites and couplings within a spin system. Each :ref:`simulator_api` object
-holds a list of :ref:`spin_sys_api` objects which will be used to calculate frequency
-contributions.
+At the heart of any ``mrsimulator`` calculation is the definition of
+a :ref:`spin_sys_api` object describing the sites and couplings within a spin
+system. Each :ref:`simulator_api` object holds a list of :ref:`spin_sys_api`
+objects which are used to calculate frequency contributions.
+
+.. note::
+
+    ``mrsimulator`` faces the same limitation faced by all other NMR simulation
+    codes: the computational cost increases exponentially(?) with the number of
```

Comment:
@pjgrandinetti @deepanshs Is there a reference for the exponential complexity?

Reply:
exponential is fine.  we can remove the question mark.